app/controllers/repair_requests_controller.rb

```diff
@@ -61,54 +43,89 @@ def create
 
   private
 
-  # FIXME: work_orders and tasks are mixed up
   def issue_dlo_work_orders
-    n_errors = 0
-    @repair_request.work_orders.each do |work_order|
-      if work_order.is_dlo?
-        begin
-          HackneyAPI::RepairsClient.new.post_work_order_issue(
-            work_order.reference,
-            created_by_email: current_user_email
-          )
-        rescue HackneyAPI::RepairsClient::ApiError => e
-          n_errors += 1
-        end
-      end
-      # FIXME: quick dirty fix. Must split work_orders and tasks anyway.
-      break
+    task = @repair_request.tasks.first
+    if task.is_dlo?
+      HackneyAPI::RepairsClient.new.post_work_order_issue(
+        task.work_order_reference,
+        created_by_email: current_user_email
+      )
     end
 
-    if 0 < n_errors
-      flash.now[:error] ||= []
-      flash.now[:error] << "There was an error issuing the work order and it
-      may not appear in DRS. This can still be done through UH. If the problem
-      persists, please contact the development team."
+  rescue HackneyAPI::RepairsClient::ApiError => e
+    flash.now[:error] ||= []
+    flash.now[:error] << t(".error_issuing")
```

Comment:
where is the `t` coming from?

Reply:
I don't know exactly, but it's Rails stuff.

Reply:
It's the same as on the views.

Reply:
> where is the `t` coming from?

@cdccollins https://guides.rubyonrails.org/i18n.html#lazy-lookup